frontend_app_workspace_advanced/src/css/index.styl

```diff
@@ -25,7 +25,10 @@
   &__description
     &__text
       padding standardSpacing
+      position relative
+      height 13em
       &__textarea
+        height 13em
```

Comment:
Why `em`?

Reply:
Easier to reason about, I changed to 200px.